ui/selectors/confirm.ts

```diff
@@ -7,6 +7,7 @@ import { getPendingApprovals } from './approvals';
 type Confirmation = {
   chainId?: string;
   id: string;
+  isScrollToBottomNeeded?: boolean;
```

Comment:
`isScrollToBottomNeeded` can be derived from hook `useScrollRequired`, I think it can be removed from state.

Reply:
currently, I'm learning to keep `isScrollToBottomNeeded`. the `useScrollRequired` hook does not have context of the confirmation(s). To rely on the values from `useScrollRequired`, we might need to move the hook up to the ConfirmPage component and pass values down to the ConfirmScrollToBottom. Is this what you were thinking? cc: @jpuri